stories/topics/proc-aap-gcp-application-upgrade.adoc

```diff
@@ -0,0 +1,88 @@
+[id="proc-aap-gcp-application-upgrade"]
+
+Technical Preview
+~~~~~~~~~~~~~~~~~
+
+You can upgrade your existing {AAPonGCP} deployment to the newer version. The upgrade process covers upgrading of automation hub and automation controller. The upgrade process takes roughly the same amount of time as a {AAPonGCP} deployment install. Follow the steps below to ensure a smooth upgrade process.
+
+[NOTE]
+=====
+{AAPonGCP} supports sequential upgrades. All upgrades should be no more than one major version behind what you are currently upgrading to. For example, in order to upgrade to {AAPonGCP} to 2.2.20230215, you must be on the first version of {AAPonGCP}.
+=====
+
+.Prerequisites
+* You require a docker installed in order to run the upgrade playbook.
+
+== Upgrade Process
+
+The following instructions describe how to run the upgrade playbook.
+
+.Procedure
+. Gather the following input variables to create an `extra_vars.yaml` file.
+.. *gcp_service_account_credentials_json_path*: Service account crendential json file path for the google cloud account.
+.. *gcp_deployment_name*: Deployment name which you want to upgrade.
+.. *gcp_compute_region*: GCP region which you provided when deploying foundation deployment. If you haven't provided region when deploying foundation, use default region here which is `us-east1`.
+.. *gcp_compute_zone*: GCP zone which you provided when deploying foundation deployment. If you haven't provided zone when deploying foundation, use default zone here which is `us-east1-b`.
++
```

Comment:
<add note here. You can discover the region and zone by checking the Filestore instance>